ai/question.py

```diff
@@ -57,16 +57,19 @@ def ask(repo_id: int, chat_id: str, question: str):
 
     memory = _get_chat_memory(chat_id)
 
-    qa = ConversationalRetrievalChain.from_llm(
-        llm=ChatOpenAI(temperature=0),
+    agent = ConversationalRetrievalChain.from_llm(
+        llm=ChatOpenAI(temperature=0, model='gpt-3.5-turbo-16k'),
         memory=memory,
         retriever=retriever,
         return_source_documents=True,
     )
 
-    result = qa(question)
+    result = agent({'question': question, 'chat_history':[]})
```

Comment:
@aaa006bd Wouldn't this be an empty chat_history? Why do you need to pass the question and chat_history here explicitly. From the docs it looked like passing `memory=memory,` line 62 to `ConversationalRetrievalChain.from_llm(` would be enough.

Reply:
@anbraten yeah you are right. Updated it